src/App.jsx

```diff
@@ -14,70 +15,88 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [valueGood, SetValueGood] = useState('Jam');
```

Comment:
try to to find better name like `selectedProduct`

when the other developer will read your code he has to undersatnd what is it